### ISSUE TITLE
Code review peer to peer week 4

### ISSUE THREAD

**Issue by fasaldana** [fed1k/Personal-Portfolio.github.io#24]
For today's coding review there are some suggestions for having a cleaner code that should be checked:
- Remove commented code from HTML file.
- Decorative images should have a null or empty decorative text for accessibility purposes.
- Adding hover effects to the card's buttons at the work section.

Everything else looked good and have nothing to be suggested.
